src/neptune_optuna/impl/__init__.py

```diff
@@ -574,10 +574,14 @@ def _log_plots(
                     vis.plot_slice(study, target=target, target_name=target_name)
```

Comment:
CHANGELOG; in addition, could you bump the `isort` version? The same updates as here: https://github.com/neptune-ai/neptune-client/commit/6e8d00c5c92faa208de8bec7cba1415066a6db68

Reply:
Done 👍